en/other/faq/technical.md

```diff
@@ -500,6 +500,14 @@ This is useful for cases like: merchant’s first request was resulting in a den
 
 Else, your email address may **have been unreachable sometimes in the past** (due to some reason explained above) when Midtrans tried to send email. When Midtrans fails to send emails about 3-5 times to an address, that address will be **marked as "unreachable/bad-address"** and Midtrans will stop sending any future emails. You will need to [contact Midtrans](https://midtrans.com/contact-us), to reset it.
 
+#### How to test offline installment on Sandbox environment?
+Offline Installment is the type of payment where Card Issuing Bank used for making an installment payment and the Acquiring Bank need not be the same. For example, a customer makes an installment payment using BNI Card and the Acquiring Bank is Mandiri.
+test offline installment on Sandbox environment
+If you need to test offline installment on Sandbox environment. You can use a value BIN dummy card e.g. `“481111, 521111”`, [here is list testing dummy card](/en/technical-reference/sandbox-test.md#card-payments). <br>
```

Comment:
Becareful when copying from other text-editor, quote chars can change. Replace with a proper JSON compatible quote `"` instead of `“`

Reply:
Rephrase this, this is confusing
> You can use a value BIN dummy card e.g. `“481111, 521111”`, [here is list testing dummy card](/en/technical-reference/sandbox-test.md#card-payments)
Some idea:
> You can use for example `"481111, 521111"` as dummy BINs value, which have corresponding test cards that are acceptable on Sandbox environment. You can check here for more sandbox test cards that you can use as BINs value. E.g. choose some cards, and then use their first 6-digit as BINs value.